### DISCUSSION TITLE
How do I use `align`?

### DISCUSSION THREAD

**Discussion by Moosems** [tukaan/tukaan#150] [Question]
I couldn't find a way to access the `Align` class nor do I really understand what it does. It can be a 0, 1, or 2 and 2 is supposed to stretch it but that doesn't happen for me. Came across this when trying to put a Label in the bottom right corner to see that the tooltip automatically went somewhere other than below or to the right but instead learned I didn't actually understand how to use the `align` argument. 
<img width="670" alt="Screenshot 2024-04-25 at 9 03 18 PM" src="https://github.com/tukaan/tukaan/assets/95927277/bc7cdb50-d3de-40cc-aa67-9d006c4bf930">

**Comment by rdbende** [comment]
`from tukaan.enums import Align`

It doesn't work wit plain numbers, and also,yu should use a tuple when providing two values.

**Reply by Moosems** [reply]
<img width="661" alt="Screenshot 2024-04-26 at 8 17 33 AM" src="https://github.com/tukaan/tukaan/assets/95927277/8f2bb224-c389-4676-98fd-519efdd91eea">

Still it's not stretching :(

**Reply by rdbende** [reply]
That's still not how you use an enum. It's `Align.Start` or `Align.Stretch` or whatever.
I think you also need to do a columnconfigure on the window, but I don't know how exactly that's done from the top of my head.
